### DISCUSSION TITLE
When you do stock adjustmen and selected delete, increment the stock

### DISCUSSION THREAD

**Discussion by ferventurart** [Blair2004/NexoPOS#270] [Gastro 4.x Ideas & Issues] [answered]
![image](https://user-images.githubusercontent.com/16241095/130142078-eb1c0861-99c6-48fe-8e46-ec09474f6cb2.png)
the reason is because you are only considerating the scenario when the action is STOCK_REDUCE

![image](https://user-images.githubusercontent.com/16241095/130142106-d9f498f3-2eb1-4254-8006-34ae3b2bb22c.png)

**Comment by ferventurart** [comment]
![image](https://user-images.githubusercontent.com/16241095/130142522-36e03cb5-76ce-4c12-97a6-fe5102eec229.png)
root of the issue

**Reply by Blair2004** [reply]
Hi, 
Yes, this seems to be a bug.

Thank you for reporting.

**Reply by emanderson2k8** [reply]
When is this going to be fixed please?

**Reply by Blair2004** [reply]
A recent commit solves this issue. https://github.com/Blair2004/NexoPOS-4x/commit/d734fd1d52db6f87b606b25db0a9e5e9ba84f2f0

**Comment by Blair2004** [comment, marked as answer]
I believe the issue has been solved already, since version 4.5.6
